Задача 4. Даны 3 группы учеников плавания. Не соблюдается условие нормальности.
В 1 группе время на дистанцию 50 м составляют:
56, 60, 62, 55, 71, 67, 59, 58, 64, 67
Вторая группа : 57, 58, 69, 48, 72, 70, 68, 71, 50, 53
Третья группа: 57, 67, 49, 48, 47, 55, 66, 51, 54
Есть ли статистически значимые различия между группами?

In [1]:
import numpy
import scipy.stats as stats
x1=[56, 60, 62, 55, 71, 67, 59, 58, 64, 67]
x2=[57, 58, 69, 48, 72, 70, 68, 71, 50, 53]
x3=[57, 67, 49, 48, 47, 55, 66, 51, 54]
a=0.05
stats.kruskal(x1,x2,x3)


KruskalResult(statistic=5.465564058257224, pvalue=0.06503809985904942)

pvalue=0.065 > a =0.05 - статистически значимые различия между группами нет( гипотеза Но  не отвергается), в группах плавают всреднем одинаково